### ISSUE TITLE
Unread Badge?

### ISSUE THREAD

**Issue by lexluthor5** [chrisknepper/android-messages-desktop#57]
I like the work you are doing with this.  But, when I just use a chrome shortcut, I get a an unread badge over the icon in my system tray if I have an unread message.  Is there a way for you to do that?  That's a deal breaker for me.

**Comment by chrisknepper** [comment]
Hi @lexluthor5 , thanks for reporting this! I'm assuming you mean on Windows. Would you mind taking a screenshot of what the unread badge in the tray looks like? I think it's definitely possible.

**Comment by lexluthor5** [comment]
That's what it looks like using the chrome shortcut minimize to the system
tray (I use 4t tray minimizer, but that has nothing to do with the icon).

**Comment by chrisknepper** [comment]
I don't think you can attach screenshots using email reply, you'll have to put the image through the GitHub website.

**Comment by lexluthor5** [comment]
Here you go.
![image](https://user-images.githubusercontent.com/41312611/42854777-889ae432-8a0b-11e8-9892-3355488fdbe1.png)

**Comment by chrisknepper** [comment]
This definitely seems possible! I will attempt to get this in an upcoming release.

**Comment by lexluthor5** [comment]
That's great.  I'll probably start using this over the chrome shortcut once this is in place.

**Comment by lexluthor5** [comment]
This is working well, thanks!
